project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/docgen_from_Jira.yaml

```diff
@@ -0,0 +1,61 @@
+name: docgen from Jira
+on:
+  workflow_dispatch
+
+jobs:
+   generate:
+     runs-on: ubuntu-latest
+     steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # full history for metadata
+          submodules: false
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Python install
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install "lander<2.0.0"
+
+      - name: condaenv
+        run: |
+          conda init bash
+          conda create --name docsteady-env docsteady -c lsst-dm -c conda-forge
+        shell: bash
+
+      - name: docugen
+        env:
+          JIRA_USER: {% raw %}${{ secrets.JIRA_USER }} {% endraw %}
+          JIRA_PASSWORD: {% raw %}${{ secrets.JIRA_PASSWORD }} {% endraw %}
+        run: |
+          source ${CONDA}/bin/activate docsteady-env
+          docsteady generate-tpr {{ cookiecutter.plan }}  {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex
+          echo `ls -la {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex`
```

Comment:
I don't think this is necessary.  If you want to keep it, can't you remove the `echo` and the backticks?